specification/windowsiot/resource-manager/readme.az.md

```diff
@@ -2,23 +2,44 @@
 
 These settings apply only when `--az` is specified on the command line.
 
-``` yaml $(az)
+``` yaml $(az) && $(target-mode) != 'core'
 az:
-  extensions: windowsiotservices
+  extensions: windows-iot-services
   namespace: azure.mgmt.windowsiot
   package-name: azure-mgmt-windowsiot
 az-output-folder: $(azure-cli-extension-folder)/src/windowsiot
-python-sdk-output-folder: "$(az-output-folder)/azext_windowsiotservices/vendored_sdks/windowsiot"
+python-sdk-output-folder: "$(az-output-folder)/azext_windowsiot/vendored_sdks/windowsiot"
+extension-mode: experimental
```

Comment:
FYI. the extension mode will be experimental by default for cli extension generation